src/os/bluestore/BlueStore.cc

```diff
@@ -2696,6 +2695,14 @@ BlueStore::BlueStore(CephContext *cct, const string& path)
     Finisher *f = new Finisher(cct, oss.str(), "finisher");
     finishers.push_back(f);
   }
+
+  for (uint32_t i =0; i < cct->_conf->bluestore_num_kv_sync_threads; i++) {
```

Comment:
Suggest to make a specific function to initialize kv_sync_threads and avoid copy/paste this way.